torchsde/_core/methods/srk.py

```diff
@@ -63,38 +59,31 @@ def diagonal_or_scalar_step(self, t0, t1, y0):
         dt = t1 - t0
         sqrt_dt = torch.sqrt(dt) if isinstance(dt, torch.Tensor) else math.sqrt(dt)
         I_k, I_k0 = self.bm(t0, t1, return_U=True)
-        I_kk = [(delta_bm_ ** 2. - dt) / 2. for delta_bm_ in I_k]
-        I_kkk = [(delta_bm_ ** 3. - 3. * dt * delta_bm_) / 6. for delta_bm_ in I_k]
+        I_kk = (I_k ** 2 - dt) / 2.
+        I_kkk = (I_k ** 3 - 3. * dt * I_k) / 6.
```

Comment:
Can these divisions be substituted for multiplications? (Making `_r6 = 1/6` a global constant is a little cleaner; writing out `0.16666...67` to full precision is a little faster; IMO doesn't really matter which is done.)

Reply:
Done.